### PR TITLE
Modified Loadout & added ItemSelection

### PR DESCRIPTION
**Pull request by AnthonyPython**

### CODE REVIEW

**Review by Deathreus — CHANGES_REQUESTED** [review]
There are many inconsistencies that need to be addressed

**Comment by AnthonyPython** [comment]
> 
> 
> There are many inconsistencies that need to be addressed

What inconsistencies, comment on what needs to be fixed.

File: src/game/client/client_tf2vintage.vpc [hunk 1]
```diff
@@ -260,6 +260,8 @@ $Project "Client (TF)"
 					$File	"tf\vgui\panels\tf_backgroundpanel.h"
 					$File	"tf\vgui\panels\tf_dialogpanelbase.cpp"
 					$File	"tf\vgui\panels\tf_dialogpanelbase.h"
+					$File	"tf\vgui\panels\tf_ItemSelectionpanel.cpp"
```
Comment (Deathreus) [inline src/game/client/client_tf2vintage.vpc]: Capitalization

File: src/game/client/tf/vgui/panels/tf_ItemSelectionpanel.cpp [hunk 2]
```diff
@@ -0,0 +1,584 @@
...
+#include "script_parser.h"
+#include "econ_item_view.h"
+
+using namespace vgui;
+// memdbgon must be the last include file in a .cpp file!!!
+#include "tier0/memdbgon.h"
+
+#define PANEL_WIDE 110
+#define PANEL_TALL 70
+CTFWeaponSelectPanel *m_weaponbutton;
```
Comment (Deathreus) [inline src/game/client/tf/vgui/panels/tf_ItemSelectionpanel.cpp]: Learn the specifics of Hungarian Notation

File: src/game/shared/tf/tf_inventory.h [hunk 3]
```diff
@@ -24,10 +24,17 @@
 
 #define INVENTORY_WEAPONS		62
 #define INVENTORY_WEAPONS_COUNT	63
-#define INVENTORY_COLNUM		63
+#define INVENTORY_COLNUM		1
 #define INVENTORY_ROWNUM		6
 #define INVENTORY_VECTOR_NUM	INVENTORY_COLNUM * INVENTORY_ROWNUM
 
+//Item Selection Def's
+#define INVENTORY_WEAPONS_Selection		62
```
Comment (Deathreus) [inline src/game/shared/tf/tf_inventory.h]: Note the rest of the constant defines in this entire solution

**Review by Deathreus — CHANGES_REQUESTED** [review]

File: src/game/client/client_tf2vintage.vpc (hunk 1 above)
Comment (AnthonyPython) [reply src/game/client/client_tf2vintage.vpc]: It isn't sensitive to a file with caps, or no?

**Review by AnthonyPython — COMMENTED** [review]
Any thing else?

File: src/game/client/tf/vgui/panels/tf_loadoutpanel.cpp [hunk 4]
```diff
@@ -14,10 +14,10 @@ using namespace vgui;
 
 #define PANEL_WIDE 110
 #define PANEL_TALL 70
-CTFWeaponSetPanel *m_weaponbutton;
-CTFWeaponSetPanel *m_weaponbutton2;
-CTFWeaponSetPanel *m_weaponspace;
-bool ITEMMENU = 0;
+CTFWeaponSetPanel *ptr_weaponbutton;
```
Comment (Deathreus) [inline src/game/client/tf/vgui/panels/tf_loadoutpanel.cpp:17]: `s_p`, camel case nitpick

Comment (AnthonyPython) [reply src/game/client/tf/vgui/panels/tf_loadoutpanel.cpp:17]: ....

**Review by Deathreus — APPROVED** [review]

**Pull request merged**